powershell/tests/functions/Invoke-Maester.Tests.ps1

```diff
@@ -3,5 +3,30 @@
         if (Get-MgContext) { Disconnect-Graph } # Ensure we are disconnected
         { Invoke-Maester } | Should -Throw 'Not connected to Microsoft Graph.*'
     }
+
+    It 'Validates smoke test results' {
+        if (Get-MgContext) { Disconnect-Graph } # Ensure we are disconnected
+
+        $maesterParams = @{
+            Path = [System.IO.Path]::GetFullPath((Join-Path (Join-Path $PSScriptRoot "..") "smoketests"))
```

Comment:
`Join-Path $PSScriptRoot ".."` makes these lines complicated. It can be easier to follow if you use `Split-Path -Path $PSScriptRoot -Parent`.

e.g., `Path = [System.IO.Path]::GetFullPath((Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath "smoketests"))`